package.json

```diff
@@ -30,7 +30,7 @@
   "dependencies": {
     "@babel/standalone": "^7.4.5",
     "he": "^1.2.0",
-    "marked": "^0.3.12"
+    "marked": "^0.7.0"
```

Comment:
`marked ` already has `0.8.0` in 3 months ago
https://www.npmjs.com/package/marked